packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -68,6 +68,310 @@ export const studioLocaleStrings = {
+
+  /** Label for filter search tooltip based on the field name */
+  'navbar.search.filter-field-name': 'Field name',
+
+  /** Label for filter search tooltip based on the field description */
+  'navbar.search.filter-field-description': 'Field description',
+
+  /** Message for showing that it's used in specific document types (used within the filter search tooltip) */
+  'navbar.search.used-in-document-types': 'Used in document types',
+
+  /** Label for "all fields" in the filter menu */
+  'navbar.search.all-fields': 'All fields',
+
+  /** Label for "shared fields" in the filter menu */
+  'navbar.search.shared-fields': 'Shared fields',
+
+  /** Label for "unknown types" in the filter menu */
+  'navbar.search.unknown-type-label': 'Unknown type',
+
+  /** Label for boolean filter - true */
+  'navbar.search.true': 'True',
+
+  /** Label for boolean filter - false */
+  'navbar.search.false': 'False',
+
+  /** Label for number and string filter */
+  'navbar.search.value': 'Value',
+
+  /** Label for number range filter - min placeholder */
+  'navbar.search.min-value': 'Min value',
+
+  /** Label for number range filter - max placeholder */
+  'navbar.search.max-value': 'Max value',
+
+  /** Label for search value in a range of numbers */
+  'navbar.search.number-items-range': `{{min}} → {{max}} items`,
+
+  /** Title label for when no search results are found */
+  'navbar.search.no-results-title': 'No results found',
+
+  /** Description for when no search results are found */
+  'navbar.search.no-results-description': 'Try another keyword or adjust your filters',
+
+  /** Title label for when the search has found an error */
+  'navbar.search.error.something-went-wrong-title': 'Something went wrong while searching',
+
+  /** Description label for when the search has found an error */
+  'navbar.search.error.something-went-wrong-description':
+    'Please try again or check your connection',
+
+  /** Label for when the asset source is unknown (used in the nav bar search, specifically filter) */
+  'navbar.search.error.unknown-asset-source': 'Unknown asset source found',
+
+  /** Title for error when no valid asset sources found */
+  'navbar.search.error.no-valid-asset-source-title': 'No valid asset sources found.',
+
+  /** Description for error when no valid asset source is found, describes that only the default asset is supported */
+  'navbar.search.error.no-valid-asset-source-only-default-description':
+    'Currently, only the default asset source is supported.',
+
+  /** Description for error when no valid asset source is found, describes that you should cehck the the current studio config */
+  'navbar.search.error.no-valid-asset-source-check-config-description': `Please ensure it's enabled in your studio configuration file.`,
+
+  /** Title for error when a filter cannot be displayed */
+  'navbar.search.error.display-filter-title': 'An error occurred whilst displaying this filter.',
```

Comment:
Not an english expert, but since we are using `while` in this `'navbar.search.error.something-went-wrong-title': 'Something went wrong while searching',` - maybe use it here as well?

Reply:
I copied the text directly to what the messaging was there previously. I don't have a strong opinion on this so I'm all for changing one way or another :)